lib/Graphs/PAG.cpp

```diff
@@ -608,7 +608,7 @@ NodeID PAG::getGepObjNode(const MemObj* obj, const LocationSet& ls)
     NodeID base = getObjectNode(obj);
 
     // Base and first field are the same memory location.
-    if (FirstFieldEqBase && ls.getOffset() == 0) return base;
```

Comment:
To be safe, I would suggest making `getOffset()` as a protected method in `SymbolTableInfo` to prevent from future usage?

Or maybe you can just use your flag inside `getOffset()`.